lib/sidekiq_jobs.rb

```diff
@@ -0,0 +1,180 @@
+require "services"
+require "sidekiq"
+require "loader/file_loader"
+require "loader/cluster_loader"
+require "loader/ht_item_loader"
+require "loader/holding_loader"
+require "loader/shared_print_loader"
+require "reports/commitment_replacements"
+require "reports/etas_organization_overlap_report"
+require "reports/uncommitted_holdings"
+require "reports/rare_uncommitted"
+require "concordance_processing"
+require "ocn_concordance_diffs"
+require "reports/compile_cost_reports"
+require "reports/compile_estimated_IC_costs"
+require "reports/compile_member_counts_report"
+require "shared_print/updater"
+require "shared_print/replacer"
+require "shared_print/deprecator"
+
+# Don't want to do this by default when we aren't running under sidekiq
+Services.register(:logger) { Sidekiq.logger }
+
+module Jobs
```

Comment:
I don't love the parallel class hierarchy with slightly different names, BUT the advantage to this is that we don't have to really change our existing code or tie it to sidekiq in any way -- only phctl and this need to know about sidekiq.

Reply:
I don't understand the nested modules here.

Reply:
Question of how to add additional jobs -- worth documenting. Probably worth additional cleanup tickets to get the remaining jobs in a common shape (probably either a 'load' shape or a 'run' shape)

Reply:
Worth documenting some of the intention here.